app/models/follow_recommendation.rb

```diff
@@ -11,6 +11,7 @@
 
 class FollowRecommendation < ApplicationRecord
   self.primary_key = :account_id
+  self.table_name = :global_follow_recommendations
```

Comment:
Should we rename the model as well, of is this too risky, or to be done in a later version?

Reply:
We may want to rename the model as well, but I wanted to have the least number of changes here.